src/Workspaces/Core/Portable/Storage/SQLite/v2/SQLitePersistentStorage_FlushWrites.cs

```diff
@@ -59,9 +59,6 @@ private void FlushWritesOnClose()
                     // read/write after releasing us.
                     _shutdownTokenSource.Cancel();
                 }, CancellationToken.None);
-
-            // Wait for that task to finish.
-            writeTask.Wait();
```

Comment:
we always had a blockign TPL call both before and after this change.  We now have the benefit though that in the DisposeAsync flow, there will be no blocking wait.

Reply:
Yeah this is good